src/App.js

```diff
@@ -3,17 +3,18 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 import About from './components/support';
 import Home from './components/home';
 
-/*
-The principal component for housing sub-components (like home, about and
-other pages) for easy navigation
-*/
+
+
+//The principal component for housing sub-components (like home, about and
+//other pages) for easy navigation
+
```

Comment:
The multiline commenting with `/***/` is correct, change it back to that

Reply:
okay